docs/Running/Publish.md

```diff
@@ -0,0 +1,83 @@
+# Publishing and deploying applications via Enarx
```

Comment:
This documentation should be versioned. As we are post-release, we will need to use an override until the next upcoming release. To do this, create a directory called "override" in the root and put the documents there (as well as please commit this to the enarx repository under docs for the upcoming release).